src/ArrayCache.php

```diff
@@ -3,11 +3,16 @@
 namespace React\Cache;
 
 use React\Promise;
+use React\Promise\PromiseInterface;
 
 class ArrayCache implements CacheInterface
 {
     private $data = array();
 
+    /**
+     * @param $key
+     * @return PromiseInterface
+     */
```

Comment:
I think all doc blocks can be removed from `ArrayCache` as they are inherited from `CacheInterface`.